demisto_sdk/commands/generate_test_playbook/test_playbook_generator.py

```diff
@@ -271,21 +278,57 @@ def create_automation_task_and_verify_outputs_task(test_playbook, command, item_
         test_playbook.add_task(task_verify_outputs)
 
 
+def get_command_examples(commands_file_path) -> dict:
+    """
+    Gets the command examples from command file with their arguments.
+
+    Args:
+        commands_file_path: command file or the content of such file.
+        specific_commands: commands specified by the user.
+
+    Return:
+        dict. Arguments separated by the commands.
+    """
+    command_examples = []  # type: list
+
+    if os.path.isfile(commands_file_path):
+        with open(commands_file_path, 'r') as examples_file:
+            command_examples = examples_file.read().splitlines()
+    else:
+        print('Failed to open command examples file.')
+        command_examples = commands_file_path.split('\n')
+
+    # Split the command example to dictionary of arguments for each command
+    result_commands = {}
+    for command in command_examples:
+        command = command.split(' ', 1)
+        result_commands[command[0].strip('!')] = dict(arg.split('=') for arg in command[1].split(' ')) \
+            if len(command) > 1 else {}
+
+    return result_commands
+
+
 class PlaybookTestsGenerator:
     def __init__(self, input: str, output: str, name: str, file_type: str, no_outputs: bool = False,
-                 verbose: bool = False, use_all_brands: bool = False):
+                 verbose: bool = False, use_all_brands: bool = False, commands: str = None, examples: str = None,
+                 upload: bool = False):
         self.integration_yml_path = input
-        self.output = output
-        if output:
-            self.test_playbook_yml_path = os.path.join(output, name + '.yml')
+
+        if not output or not os.path.isdir(output):
+            self.output = f'Packs/{get_pack_name(self.integration_yml_path)}/TestPlaybooks'
```

Comment:
this is not good cause we might not be running from content repo here.
you can either:
1) check if we are running from content and then use this path. (use `is_external_repository`)
2) if we are not in content just put the file locally.

Also what if I'm running in content in a pack that does not have TestPlaybooks dir? shouldn't we need to create it before putting the full path?

Reply:
fixed.